packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -126,11 +126,12 @@ export async function clearAuthenticationState( page: Page ): Promise< void > {
 	const currentURL = page.url();
 
 	// Navigate to the WordPress.com base URL.
-	await page.goto( 'https://r-login.wordpress.com/' );
+	await page.goto( 'https://r-login.wordpress.com/', { waitUntil: 'networkidle' } );
 	// Clear local storage.
 	await page.evaluate( 'localStorage.clear();' );
 	// Lastly, clear the cookies using built-in method.
 	await browserContext.clearCookies();
+	await page.waitForTimeout( 1000 );
```

Comment:
Debugging leftovers?

Reply:
No, as noted in the description I've added the delay timeout like what the Selenium suite has.

There have been intermittent failures on CI with this step, and I suspect it needs a bit of time to settle.

Reply:
Yeah, I think this should be dealt with separately. I'm sure there's a better way of handling this than via a fixed timeout. I've had a really bad experience with Selenium's fixed _waits_ as they have never worked well as a long-term solution.

Reply:
I've removed this section, will spin up a different PR to address making `clearAuthenticationState` method more reliable.